docs/backup_workstations.rst

```diff
@@ -118,16 +115,90 @@ Then, copy the contents of the device's persistent volume to the directory using
 
 .. code:: sh
 
-  rsync -a --info=progress2 --no-specials --no-devices \
-      /media/amnesia/TailsData/ /media/amnesia/Backup/admin-backup
+  sudo bash -c "rsync -a --info=progress2 --no-specials --no-devices \
+      /media/amnesia/TailsData/ /media/amnesia/Backup/admin-backup && sync"
+
+.. note:: Note that the first path argument ends with a ``/``, but the second 
+          path argument should not!
 
-Once complete, unmount the TailsData partition.
+
+Once complete, unmount the TailsData partition by clicking the Eject button 
+beside its entry in the lefthand column of the file manager. When its entry is 
+no longer shown in the lefthand column, it is save to remove the 
+*Admin Workstation* USB.
 
 Repeat these steps for every device, making a new folder on the backup device
 for each device you back up.
 
 Finally, once you have completed the steps described in this section for each
-USB drive, unmount the Backup partition and store the drive somewhere safely.
+USB drive, unmount the Backup partition by clicking its Eject button. Wait until
+the Backup USB can be safely removed, and store it somewhere safely.
+
+.. note:: After the Eject button is clicked, it may be take some time before 
+          the drive can be safely removed. Wait until its entry  is removed from
+          the lefthand column of the file manager.
+
+Restoring a Workstation from a Backup
+=====================================
+
+To recreate a backed-up *Admin Workstation*, *Journalist Workstation*, or 
+*Secure Viewing Station* Tails USB,  you will need 
+
+- your Backup USB containing the persistent volume to be restored,
+- a fresh USB stick to be set up as the new Tails USB,  
+- an airgapped machine and a USB with Tails already installed, referred to as 
+  the host Tails USB in this document. The host Tails USB is only used to 
+  transfer files between the Backup USB and the new Tails USB.
```

Comment:
Should we say something here about the persistent volume of the host USB, or would that make things more confusing? For example:

> It does not require a persistent volume; if one is configured, you do not need to unlock it during this process.

I'm thinking it might help to give folks peace of mind that they're not accidentally copying to the wrong persistent volume.

Reply:
In order to do so, you'd need to use a completely different path, as the PV is mounted under `/live/persistence/` rather than under the default location for attached storage in `/media/amnesia,' so my feeling is that this is probably not worth the added complexity. There's no mention of the PV in the step where the USB is started and it wouldn't be a problem if they did unlock it anyway.

Reply:
:+1: